build-system/tasks/extension-generator/doc-template.md

```diff
@@ -0,0 +1,76 @@
+<!--
+  1. Change "category" below to one of:
+       ads-analytics
+       dynamic-content
+       layout
+       media
+       presentation
+       social
+  2. Remove any of the "formats" that don't apply.
+     You can also add the "ads" and "stories" formats if they apply.
+  3. And remove this comment! (no empty lines before "---")
+-->
+---
+$category: presentation
+formats:
+  - websites
+  - email
+teaser:
+  text: FILL THIS IN.
+---
+<!--
+Copyright ${year} The AMP HTML Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# \`${name}\`
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>FILL THIS IN</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Availability</strong></td>
+    <td>FILL THIS IN</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Script</strong></td>
+    <td><code>&lt;script async custom-element="${name}" src="https://cdn.ampproject.org/v0/${name}-0.1.js">&lt;/script></code></td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><strong><a href="https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/control_layout">Supported Layouts</a></strong></td>
```

Comment:
`s/fourty/forty/` ?

Reply:
Colonel Fourty? :smiley:

Reply:
Lol, this is hilarious, but I think amp.dev relies on this naming convention for now... I also think Sebastian is planning a new process to help with the creation of the tables.